api/src/pcapi/routes/backoffice/admin/blueprint.py

```diff
@@ -129,3 +130,13 @@ def toggle_feature_flag(feature_flag_id: int, set_to_active: bool) -> utils.Back
         f"Le feature flag {feature_flag.name} a été " + ("activé" if feature_flag.isActive else "désactivé"), "success"
     )
     return redirect(url_for(".list_feature_flags"), code=303)
+
+
+@blueprint.backoffice_web.route("/admin/category-subcategory", methods=["GET"])
```

Comment:
```suggestion
@blueprint.backoffice_web.route("/admin/subcategories", methods=["GET"])
```
Car au final, tu listes les sous-catégories seulement

Reply:
`@blueprint.backoffice_web.route("/admin/subcategories", methods=["GET"])`
FAIT